package.json

```diff
@@ -22,29 +26,34 @@
     "@angular/platform-browser": "^6.0.3",
     "@angular/platform-browser-dynamic": "^6.0.3",
     "@angular/router": "^6.0.3",
+    "concurrently": "^3.6.0",
     "core-js": "^2.5.4",
+    "json-concat": "0.0.1",
+    "json-server": "^0.14.0",
+    "node-sass": "^4.9.2",
```

Comment:
Why is node-sass required for json server?

Reply:
node-sass require for project and include together of json-server commit

Reply:
As discussed, we will allow this one to be present. 
- Resolved